lms/djangoapps/mobile_api/users/views.py

```diff
@@ -410,3 +414,128 @@ def my_user_info(request, api_version):
     # updating it from the oauth2 related code is too complex
     user_logged_in.send(sender=User, user=request.user, request=request)
     return redirect("user-detail", api_version=api_version, username=request.user.username)
+
+
+class UserCourseEnrollmentsV4Pagination(DefaultPagination):
```

Comment:
Are we using it anywhere?

Reply:
This probably got here from another commit. Removed